### ISSUE TITLE
Model Training & Prediction

### ISSUE THREAD

**Issue by femijulius560** [femijulius560/healthcare_insurance_claim#8]
- [x] Select models (Random Forest & Linear Regression)
- [x] Train models on training set
- [x] Make predictions on test set
- [x] Evaluate model performance (MAE, RMSE, R²)
